### ISSUE TITLE
Support Knockout comments in HTML

### ISSUE THREAD

**Issue by rudym87** [xoofx/NUglify#8]
Currently Knockout comments for [containerless bindings](http://knockoutjs.com/documentation/if-binding.html#note-using-if-without-a-container-element) in a HTML file get stripped away like normal comments. 
A fragment like this:  
````html
<div>
    <!--ko if: observable-->
    <p></p>
    <!--/ko-->
</div>
````
gets minified to this:
````html
<div><p></p></div>
````

instead of this:
````html
<div><!--ko if:observable--><p></p><!--/ko--></div>
````

**Comment by madskristensen** [comment]
This is high priority IMO

**Comment by xoofx** [comment]
This is fixed by commit  	44a36ae
